src/GogeToken.sol

```diff
@@ -1485,4 +1433,11 @@ contract CakeDividendTracker is DividendPayingToken {
     	return false;
     }
 
+    function getMapValue(address key) public view returns (uint) {
```

Comment:
Why were these functions added? Also, any function that won't be called by this contract should be marked as `external`, not `public` to save on gas.

Reply:
fixed